### ISSUE TITLE
Html export should have touch controls

### ISSUE THREAD

**Issue by TripleCubes** [nesbox/TIC-80#2476]
I dont see a reason why tic80 html exports doesnt have touch controls. This is definitely not a tech problem as I was able to add touch controls to my web export just by recompiling tic80 wasm with touch controls enabled.

Maybe it is because we dont want the touch controls to appear for desktop users? It is also solvable by just making the touch controls not appear by default and only appear when a touch input is made. Doing this only require changing one single line of code, as I was able to do so.

**Comment by nesbox** [comment]
Touch controls were designed for use on mobile devices, and I didn't think we'd be able to use them on the web.
But if you say it works, then of course we can enable it :)

**Comment by TripleCubes** [comment]
Yes. But consider ios user who doesnt have a easy way to install tic80, tic80 web is the easiest way to be able to play tic80 games for them. And for that touch controls for the web is essential.

When running a tic80 web game in itch.io on mobile, clicking the full screen button will hide the browser bar. So it is a really good way to play tic80 games.

It is great to see the touch control will be enabled. Thank you for your good work.

**Comment by matstc** [comment]
It would be ideal if the export came with touch controls but until then, it's possible to customize the export by swapping the `index.html`. The template attached has worked well for me.

[generic_index.html](https://github.com/user-attachments/files/26189714/generic_index.html)
